src/Sulu/Bundle/TrashBundle/UserInterface/Controller/Admin/TrashItemController.php

```diff
@@ -190,12 +190,8 @@ public function cgetAction(Request $request): Response
 
         $trashItems = \array_map(
             function(array $trashItem) use ($hiddenFieldDescriptors) {
-                if (isset($trashItem['resourceKey'])) {
-                    $trashItem['resourceKey'] = $this->getResourceTranslation($trashItem['resourceKey']);
-
-                    if (!empty($trashItem['restoreType'])) {
-                        $trashItem['resourceKey'] .= ' (' . $this->getResourceTranslation($trashItem['restoreType']) . ')';
-                    }
+                if (isset($trashItem['resourceType'])) {
```

Comment:
not sure if we re testing this controller. if we do, it would be nice to test the `restoreType` too 🙂

Reply:
updated the testCgetAction to test this behaviour.